src/js/components/NavTabsComponent.jsx

```diff
@@ -1,21 +1,38 @@
 var classNames = require("classnames");
 var React = require("react/addons");
 
+var DeploymentEvents = require("../events/DeploymentEvents");
+var DeploymentStore = require("../stores/DeploymentStore");
+var DeploymentActions = require("../actions/DeploymentActions");
+
 var NavTabsComponent = React.createClass({
   displayName: "NavTabsComponent",
 
   propTypes: {
     activeTabId: React.PropTypes.string.isRequired,
+    activeDeployments: React.PropTypes.number,
```

Comment:
You can take an example in the info tests. You can override the deployments object from the store beforehand, f.ex., to have to deployments do in the test:
```
DeploymentStore.deployments = [
  {
    id: "deployment-1"
  },
  {
    id: "deployment-2"
  }
];
```